client/src/components/Home.jsx

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
-import { Row, Col } from 'react-materialize'
+import React, { Component } from 'react';
+import {Row, Col} from 'react-materialize'
+import {server_url} from '../config.json'
 
 
-function Home() {
-  return (
-    <Row>
-      <Col s={4} offset="s2"><h1>This is Home</h1></Col>
-    </Row>
-  );
-}
+export default class Home extends Component {
+  componentDidMount() {
+    fetch(server_url)
+      .then(response => {
+        return response.json()
+      })
+      .then(data => {
+        console.log('Data: ', data)
+      })
+  }
 
-export default Home;
+  render() {
+    return (
+      <Row>
+        <Col s={1}></Col>
```

Comment:
Why first <Col> is empty?

Reply:
That is a merged code from dev, have no idea myself